lib/js/src/manager/AppConfig.js

```diff
@@ -315,6 +317,44 @@ class AppConfig {
     getMinimumProtocolVersion () {
         return this._minimumProtocolVersion;
     }
+
+    /**
+     * Load manifest file of embedded Web Application
+     * @param {Object} manifest
+     * @return {AppConfig}
+     */
+    loadManifest (manifest) {
+        const manifestKeys = Object.keys(manifest);
```

Comment:
Please perform basic type validation on the `manifest` argument to ensure it is an Object before accessing Object-specific methods. If the type is not valid, an error may be thrown to alert the developer of their mistake.

Reply:
Done in 469cbdfd19c0ffbb04ef6e49635acf7d0d2be3b0